components/binance/browser/BUILD.gn

```diff
@@ -20,8 +20,12 @@ source_set("browser") {
 
   deps = [
     "//base",
+    "//brave/common:pref_names",
```

Comment:
if these are missing deps you discovered when running it, can we at least put them in a separate commit?

Reply:
moved to a separate commit.